### PR TITLE
Added schedule announcement date

### PR DESCRIPTION
**Pull request by tokt**
Based on request from slack.

### CODE REVIEW

**Comment by sftim** [comment]
@tokt, how can I  a reviewer check this change? (do you have a link to the request on Slack?)

**Comment by tokt** [comment]
@sftim Sincere apologies, should have included the link: https://kubernetes.slack.com/archives/CEMM39SKG/p1693475849985999
@nikhita Do you want any other dates to be mentioned too?

File: content/en/events/2023/kcsna/schedule.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ weight: 3
 ## CFP
 
 A [CFP] for sessions is open through Friday, September 15th.
+Please, expect the schedule to be announced on October 6th, 2023.
```
Comment (nikhita) [inline content/en/events/2023/kcsna/schedule.md:12]: Let's mention all of the dates from the email to k-dev?

https://groups.google.com/a/kubernetes.io/g/dev/c/Gg5p-iyRaF0/m/DQkndQSMAwAJ

```
    CFP Opens: Tuesday August 15th

    CFP Closes: Friday, September 15th

    CFP Notification: Friday, September 29th

    Schedule Announced: Friday, October 6th
```

**Review by jberkus — APPROVED** [review]

**Review by windsonsea — COMMENTED** [review]
/lgtm

**Pull request merged**